mac/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Schedual</title>
+	<link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+
+	<header>
+		
+		<div class = "navbar">
+	
+			<ul>
+				<div class = "logo">
+					<li><a href="#"><img src="logo.png"></a>
+					</li>
+				</div>
+
+				<div id="courseCatalog">
+					<li ><a href="http://catalog.oregonstate.edu/Default.aspx?section=Course">Course Catalog</a></li>
+				</div>
+			</ul>
+		</div>
+	
+	</header>
+	
+	<div class = "tableClassSets">
+		
+		<div><h2>ClASS SETS</h2></div>
+
+		<div class= "tableHolder">
+			<table>
+				<tr>
```

Comment:
It would be a good idea to place the header row in a `<thead>` element and the main table in a `<tbody>` element